src/gui/threejsScene.js

```diff
@@ -104,6 +113,24 @@ import { ViewFrustum, frustumVertexShader, frustumFragmentShader, MAX_VIEW_FRUST
                 toggleDisplayOriginBoxes(newValue);
             }, { dontPersist: true });
         }
+        
+        document.addEventListener('keydown', (e) => {
+            e.stopPropagation();
+            if (e.key === 'n' || e.key === 'N') {
+                navmesh.visible = !navmesh.visible;
+            }
+        })
```

Comment:
Good idea. Do you think it's worth it to add the navmesh in the scene in case someone wants to see it / debug? If not we can remove it all together.

Reply:
Hmm I'm not sure. Might be useful to see if you are getting unexpected results from the pathfinding? Has it been helpful for you to see the navmesh? If you use it a lot, maybe others would too. But if you don't really use it then maybe it's not necessary.

Reply:
I do use it a lot when the path is not giving me an obvious intuitive result. Maybe I should include it in the developer's menu, since most users shouldn't be messing around with this anyways.